### PR TITLE
HTML Editor

### PR DESCRIPTION
**Pull request by AshokPatel01**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,17 +19,20 @@
     "@angular/platform-browser": "~9.1.11",
     "@angular/platform-browser-dynamic": "~9.1.11",
     "@angular/router": "~9.1.11",
+    "bootstrap": "^4.5.0",
+    "jquery": "^3.5.1",
+    "jsoneditor": "^9.0.0",
```
Comment (mahavirvora) [inline package.json]: Remove jquery & json editor

**Review by mahavirvora — CHANGES_REQUESTED** [review]
The assets folder should be outside of src.

File: package.json (hunk 1 above)
Comment (AshokPatel01) [reply package.json]: Removed

**Comment by AshokPatel01** [comment]
Screenshot: https://www.screencast.com/t/N45iiBhriNtH

File: angular.json [hunk 2]
```diff
@@ -92,7 +92,7 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "./node_modules/bootstrap/dist/css/bootstrap.css"
```
Comment (mahavirvora) [inline angular.json]: Keep "src/styles.scss"
remove bootstrap.css from here

File: package.json [hunk 3]
```diff
@@ -16,20 +16,22 @@
     "@angular/compiler": "~9.1.11",
     "@angular/core": "~9.1.11",
     "@angular/forms": "~9.1.11",
+    "@angular/localize": "^9.1.11",
```
Comment (mahavirvora) [inline package.json]: Why are we adding this?

File: src/app/app.component.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
+@import "~bootstrap/dist/css/bootstrap.css";
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap");
```
Comment (mahavirvora) [inline src/app/app.component.scss]: Move these imports to styles.scss file

**Review by mahavirvora — CHANGES_REQUESTED** [review]

File: angular.json (hunk 2 above)
Comment (AshokPatel01) [reply angular.json]: Updated as per comment.

File: package.json (hunk 3 above)
Comment (AshokPatel01) [reply package.json]: Not required now.

File: src/app/app.component.scss (hunk 4 above)
Comment (AshokPatel01) [reply src/app/app.component.scss]: Moved in style.scss

**Pull request merged**